### PR TITLE
[component] Translation

### PR DESCRIPTION
**Pull request by ndufreche**
backbee/backbee-standard#225

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ pages:
         - 'Requirements': 'developper-documentation/requirements.md'
         - 'Architecture': 'developper-documentation/architecture.md'
         - 'Configuration': 'developper-documentation/configuration.md'
+        - 'Translation': 'developper-documentation/translation.md'
```
Comment (mickaelandrieu) [inline README.md]: should be located in Component part right ?

File: doc/developper-documentation/components/translation.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+# Components
+
+## The Translation component
+
+This component is used in BackBee to translate text into differents languages. Note this component is an extension of [Symfony Translation Component](http://symfony.com/doc/current/components/translation/index.html)
+
+
+### How the translation work
+
+BackBee translation component override Symfony\Component\Translation\Translator to lazy load every catalogs from:
```
Comment (mickaelandrieu) [inline doc/developper-documentation/components/translation.md]: ``Symfony\Component\Translation\Translator``

File: mkdocs.yml [hunk 3]
```diff
@@ -16,6 +16,7 @@ pages:
         - 'Installation': 'developper-documentation/installation.md'
         - 'Architecture': 'developper-documentation/architecture.md'
         - 'Configuration': 'developper-documentation/configuration.md'
+        - 'Translation': 'developper-documentation/translation.md'
```
Comment (mickaelandrieu) [inline mkdocs.yml]: Again, I think it is in `Components` part

**Comment by mickaelandrieu** [comment]
@ndufreche nice job, can you move these part into ``component`` section ?

**Comment by ndufreche** [comment]
yes, I do it right now.

**Comment by mickaelandrieu** [comment]
thank you, nice work @ndufreche

**Comment by ndufreche** [comment]
thanks for review

**Pull request merged**